visualization/tooltip.js

```diff
@@ -1,12 +1,4 @@
-const generateTooltipHtml = (data, checkedAttributes) => {

```

Comment:
Should we keep the raw stats? The pie chart only shows the makeup of their combined score.

Reply:
On second thought lets remove these, probably not the best to show any raw numbers in an info vis class. I've added a click event to the dots that will toggle between the pie chart and stats in another branch.